app/src/main/java/com/trueedu/project/ui/theme/Theme.kt

```diff
@@ -9,16 +9,31 @@ import androidx.compose.material3.dynamicLightColorScheme
 import androidx.compose.material3.lightColorScheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.LocalContext
+import com.trueedu.project.ui.theme2.AppTheme2
 
 private val DarkColorScheme = darkColorScheme()
 
 private val LightColorScheme = lightColorScheme()
 
 @Composable
 fun TrueProjectTheme(
+    n: Int = 1,
+    forceDark: Boolean = false,
+    content: @Composable () -> Unit
+) {
+    val darkTheme = forceDark || isSystemInDarkTheme()
+    when (n) {
+        0 -> TrueProjectThemeDefault(darkTheme, false, content)
+        1 -> AppTheme2(darkTheme, false, content)
+        else -> TrueProjectThemeDefault(darkTheme, false, content) // default
+    }
```

Comment:
인자가 모두 동일하군요. 요렇게 표현할 순 없나요?

```suggestion
    val theme = when (n) {
        0 -> TrueProjectThemeDefault
        1 -> AppTheme2
        else -> TrueProjectThemeDefault // default
    }
    theme(darkTheme, false, content)
```

`.class` 같은걸 붙여야 하려나..

Reply:
이게 일반 메소드는 reference 로 받을 수 있는데 Composable 은 그냥은 안 되는 거 같습니다.
좋은 의견 제시 ㄱㅅ합니다.